pkg/runtime/runtime.go

```diff
@@ -383,6 +385,23 @@ func (a *DaprRuntime) Run(parentCtx context.Context) error {
 	return a.runnerCloser.Run(ctx)
 }
 
+func continuouslyRetrySchedulerClient(ctx context.Context, opts clients.Options) (*clients.Clients, error) {
+	for {
+			cli, err := clients.New(ctx, opts)
+			if err == nil {
+				return cli, nil
+			}
+
+			log.Errorf("Failed to initialize scheduler clients: %s. Retrying...", err)
+select {
```

Comment:
I did the sign off and commit batch, but it looks like it messaged up the indentation here. let me fix that